packages/core/src/rules/arazzo/index.ts

```diff
@@ -19,7 +19,7 @@ export const rules: ArazzoRuleSet<'built-in'> = {
   spec: Spec as ArazzoRule,
   assertions: Assertions as ArazzoRule,
   'parameters-not-in-body': ParametersNotInBody as ArazzoRule,
-  'sourceDescription-type': SourceDescriptionType as ArazzoRule,
+  'sourceDescription-type': SourceDescriptionsType as ArazzoRule,
```

Comment:
Just rename file to have one name pattern.

Reply:
Got it, and updated it in the other pages!